src/oss.yml

```diff
@@ -127,10 +115,6 @@ components:
       $ref: "./common/schemas/Authorization.yml"
     AuthorizationPostRequest:
       $ref: "./common/schemas/AuthorizationPostRequest.yml"
-    LegacyAuthorizationPostRequest:
-      $ref: "./oss/schemas/LegacyAuthorizationPostRequest.yml"
-    Authorizations:
-      $ref: "./common/schemas/Authorizations.yml"
```

Comment:
I believe this needs put back in to resolve @bednar's [last comment](https://github.com/influxdata/openapi/pull/330#discussion_r886341763)

Reply:
Yep, looks like I had deleted that and didn't notice because I saw that `Authorization` (no s) was still there.
Adding it back in.